from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
do we need this file in the repo? I believe we generate estimate.py from the notebook.